lib/pipefy_message/broker/aws/sns/publisher.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "aws-sdk-sns"
+require "json"
+
+module PipefyMessage
+  # Aws SNS Publisher class to publish json messages into a specific topic
+  class SnsPublisher
+    def initialize
+      aws_config = PipefyMessage::AwsProviderConfig.instance
```

Comment:
We could run like this here?

`PipefyMessage::AwsProviderConfig.instance.setup_connection`